apps/meteor/tests/end-to-end/api/01-users.js

```diff
@@ -747,6 +747,43 @@ describe('[Users]', function () {
 				.end(done);
 		});
 
+		it('should sort for user statuses and check if user is correctly sorted by name', async () => {
```

Comment:
Change the test description, it is not sorting by user status.

Reply:
tkss, fixed